kornia/contrib/distance_transform.py

```diff
@@ -0,0 +1,110 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+import kornia
+
+
+def make_cdt_kernel(
+    kernel_size: int,
+) -> torch.Tensor:
+    # Value of h is derived from the parameters and reference code given by the authors who proposed the algorithm.
+    h = -0.35

```

Comment:
```suggestion
    h: float = -0.35

```

Reply:
we use also this value in the main function, possibly we could pass as function parameter

Reply:
Adding it in as one